astropy/utils/misc.py

```diff
@@ -386,7 +386,7 @@ def default(self, obj):
         import numpy as np
         if isinstance(obj, (np.ndarray, np.number)):
             return obj.tolist()
-        elif isinstance(obj, (complex, np.complex)):
+        elif isinstance(obj, (complex, np.complexfloating)):
```

Comment:
Not sure if that check is correct or if I should rather have removed the second type.

Reply:
Oh, dear. I added this in #552 to support Cone Search. I'll have to find time to dig through the 181 comments to why why I wrote this the way I did...

Reply:
Nevermind, I probably should just remove the second type to check. Currently it was only checking for `complex, complex` and `np.complexfloating` would be already catches by the `np.number` isinstance check.